nncf/quantization/algorithms/bias_correction/algorithm.py

```diff
@@ -175,17 +176,17 @@ def apply(
 
             channel_axis = node.metatype.output_channel_axis
             if current_bias.ndim > 1:
-                channel_axis = range(current_bias.ndim)[channel_axis]
-                axes = [i for i in range(current_bias.ndim) if i != channel_axis]
-                bias_shift = np.expand_dims(bias_shift, axes)
+                new_shape = [1] * current_bias.ndim
+                new_shape[channel_axis] = bias_shift.shape[0]
+                bias_shift = bias_shift.reshape(new_shape)
 
             updated_bias = current_bias + bias_shift
             magnitude = self._get_bias_shift_magnitude(current_bias, updated_bias)
 
             if magnitude < self.threshold:
                 nncf_logger.debug(f"{node_name} bias would be changed. Magnitude: {magnitude}")
                 bias_correction_command = self._backend_entity.create_bias_correction_command(
-                    node, updated_bias, nncf_graph
+                    node, updated_bias.data, nncf_graph
```

Comment:
Why can't we pass the `Tensor` into transformation and unpack the value in the `ModelTransformer`?

Reply:
Change signature to `def create_bias_correction_command(node: NNCFNode, bias_value: Tensor) -> TransformationCommand:` and unwrap data in backend part.
But not think that `ModelTransformer` should operate with Tensor, it's backend specific part. at least there is no common commands.

My suggestion, that `algorithms.py` should works with tensors, `*_backend.py` unwrap data and call backend specific functions.